src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -612,12 +650,94 @@ private void WriteObjectsOrShowOnlineHelp(HelpInfo helpInfo, bool showFullHelp)
                             }
                         }
 
-                        WriteObject(helpInfo.ShortHelp);
+                        WriteObjectPaged(helpInfo.ShortHelp, bufferOutputWhenPaged: true);
                     }
                 }
             }
         }
 
+        private void WriteObjectPaged(object sendToPipeline, bool bufferOutputWhenPaged)
+        {
+            if (Paged)
+            {
+                var helpText = GetHelpOutput();
+
+                if (bufferOutputWhenPaged)
+                {
+                    // The output from GetHelpOutput has all the item in the output.
+                    // We just need to store it and then write to Pager later.
+                    // No need to append them.
+                    _multiItemPagerBuffer = helpText;
```

Comment:
Design question about Pager - is it a common use case? If so should it be the Pager feature?

Reply:
We have had a discussion about it. We would like to get some feedback from the community if they see value in making this a built in function.

Reply:
Please do. Out-Host -Paging is probably the least useful thing. Having a native pager in PS would let folks utilize it regardless of the host OS and what external paging utils may or may not be available.

Reply:
if it relies on the alternate character buffer, then it has reduced applicability

Reply:
Can you expand on that a bit for those of us not intimately familiar with the APIs in play here? 🙂

Reply:
Opened issue #13405 to discuss if the Pager should be made more general purpose. Lets continue discussion over there.